planet_explorer/tests/test_orders.py

```diff
@@ -99,6 +103,79 @@ def test_order_download(
         qtbot.waitUntil(order_item.order.downloaded, timeout=60 * 1000)
 
 
+def test_order_add_to_map(
+    qtbot,
+    logged_in_explorer_dock_widget,
+    qgis_debug_enabled,
+    order_monitor_widget,
+    qgis_version,
+):
+    """This test is performed on the 'Add to map' button of the orders monitor widget.
+    An image is copied from the plugin directory/repo to the Planet orders directory.
+    This directory stores the downloaded orders. The widget of a particular download is
+    initilalized and added to the QGIS canvas instance. If the image could not be added,
+    the test will fail. If the image cannot be found in the map layers list of the canvas
+    after adding the image, the test will also fail.
+    """
+    dock_widget = logged_in_explorer_dock_widget()
+    order_monitor = order_monitor_widget(dock_widget)
+    dock_widget.hide()
+
+    # The test data for this function is stored here: planet_explorer/tests/Data/test_add_to_map
+    image_id = "5c9e6c59-eb35-485d-ab7d-04a75e9e0f14"  # Planet order ID
```

Comment:
We should `pytest` parametrized fixtures to pass specific params to the test function, for reference see https://github.com/planetlabs/qgis-planet-plugin/blob/592cc6f941c0715215c9e0a844c89779bf0c7331/planet_explorer/tests/test_filters.py#L10